src/WebJobs.Script/Workers/Http/HttpFunctionInvocationDispatcher.cs

```diff
@@ -65,15 +65,14 @@ internal class HttpFunctionInvocationDispatcher : IFunctionInvocationDispatcher
 
         internal Task InitializeHttpWorkerChannelAsync(int attemptCount, CancellationToken cancellationToken = default)
         {
-            // TODO: Add process managment for http invoker
-
             _httpWorkerChannel = _httpWorkerChannelFactory.Create(_scriptOptions.RootScriptPath, _metricsLogger, attemptCount);
             _httpWorkerChannel.StartWorkerProcessAsync(cancellationToken).ContinueWith(workerInitTask =>
                  {
                      if (workerInitTask.IsCompleted)
                      {
                          _logger.LogDebug("Adding http worker channel. workerId:{id}", _httpWorkerChannel.Id);
                          State = FunctionInvocationDispatcherState.Initialized;
```

Comment:
Nit: remove this line?

Reply:
hmm...deleted commented lines. No new line here.